manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,77 @@
+class icinga2::pnp4nagios::config inherits icinga2::pnp4nagios::params {
+
+  $monitoring_type = 'icinga'
+  $log_type = 'syslog'
+  $debug_lvl = 1
+  $system_date = generate('/usr/bin/date', '+%Z')
+
+  file { "$httpd_path/$htpasswd_config":
```

Comment:
here you are assuming the $httpd_path and $htpasswd_config comes from the params.pp. But what happens if the use change the service name in the pnp4nagios class? will the change in resource names gets reflected here?